config/routes.rb

```diff
@@ -14,6 +14,7 @@
 
   get 'search/', to: 'search#search'
   get 'search/tips', to: 'search#tips'
+  get 'data/search', to: 'legacy_search#redirect'
```

Comment:
@laurentqro what do you think about splitting the routes.rb so its clearer which ones are legacy?
https://blog.arkency.com/2015/02/how-to-split-routes-dot-rb-into-smaller-parts/

Reply:
Good point. 👍 

Splitting into a separate file might be premature, but we could start namespacing legacy stuff, that would already make things much clearer. Can do that in a separate PR.